secboot/secboot_tpm.go

```diff
@@ -366,7 +405,12 @@ func ResealKey(params *ResealKeyParams) error {
 		return err
 	}
 
-	return sbUpdateKeyPCRProtectionPolicy(tpm, params.KeyFile, params.TPMPolicyUpdateDataFile, pcrProfile)
+	return sbUpdateKeyPCRProtectionPolicy(tpm, params.KeyFile, params.AuthKey, pcrProfile)
+}
+
+func unlinkUserKeyring() error {
```

Comment:
this from merits a longer name:

unlinkUserKeyringFromSessionKeyring

so that Maciej's worry is clarified